crates/katana/storage/db/src/utils.rs

```diff
@@ -16,8 +16,10 @@ pub(crate) fn default_page_size() -> usize {
     os_page_size.clamp(MIN_PAGE_SIZE, LIBMDBX_MAX_PAGE_SIZE)
 }
 
-/// Check if a db is empty. It does not provide any information on the
-/// validity of the data in it. We consider a database as non empty when it's a non empty directory.
+/// Check if a Db is empty. We consider a database as non empty when `path` it's a non empty
+/// directory or the `path` exists.
```

Comment:
Not sure if this matches the actual behavior. The code returns true (empty) if the path does not exist.

Reply:
ah yes it should be ".. or if the `path` does not exist. "